editors/code/tsconfig.json

```diff
@@ -15,6 +15,7 @@
         "noFallthroughCasesInSwitch": true,
         "newLine": "LF",
         "esModuleInterop": true,
+        "importHelpers": true
```

Comment:
Please add a trailing comma to minimize diffs when adding a new line after this.

Reply:
Hmm, `JSON` protocol doesn't actually allow for trailing commas.
But, as a coincidence `tsconfig.json` doesn't follow the strict json protocol, it allows for comments and trailing commas. But I'd rather not abuse this feature anyway.

By the way, guys, if you feel not enough experienced with TypeScript and need to do some work on VSCode extension part, please assign this to me :wink:.